src/models/Plugins.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * LINKURIOUS CONFIDENTIAL
```

Comment:
Files in the REST client should not be marked as confidential.
We can keep the copyright line, but "LINKURIOUS CONFIDFENTIAL" must go

Reply:
changing it globally